db/database.go

```diff
@@ -526,6 +526,7 @@ func (context *DatabaseContext) Close() {
 	context.BucketLock.Lock()
 	defer context.BucketLock.Unlock()
 
+	context.OIDCProviders.Stop()
```

Comment:
It looks like context.OIDCProviders is going to be nil in the case where there aren't any providers defined, which seems like it will panic inside Stop().  Can you verify whether a nil check is needed either here or in Stop()?

Reply:
Sure Adam, the context.OIDCProviders is going to be nil in the case where there aren't any providers defined but it will not be panic inside Stop(); the method dispatch is determined by the type of the expression and not by the particular run-time value of that expression. [Here is a quick test](https://play.golang.org/p/vQQJchygrAl)

Reply:
Ah right - reads on a nil map are valid, and Stop is just doing read/iteration.  ok.